docs/error_pages/404.rst

```diff
@@ -0,0 +1,8 @@
+ Error 404
+===========
```

Comment:
Are the extra spaces around this heading important?  I'm not familiar with that rst construct.